x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/multi_signal_inventory/index.tsx

```diff
@@ -163,45 +166,53 @@ export function MultiSignalInventory() {
     return callApmApi('GET /internal/apm/has_entities');
   }, []);
 
-  if (!data?.hasData && status === FETCH_STATUS.SUCCESS) {
-    return <NoEntitiesEmptyState />;
-  }
   return (
     <>
-      <EuiFlexGroup gutterSize="m">
-        <EuiFlexItem grow>
-          <TableSearchBar
-            placeholder={i18n.translate('xpack.apm.servicesTable.filterServicesPlaceholder', {
-              defaultMessage: 'Search services by name',
-            })}
-            searchQuery={searchQuery}
-            onChangeSearchQuery={setSearchQuery}
-          />
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <SearchBar showQueryInput={false} />
-        </EuiFlexItem>
-      </EuiFlexGroup>
-      <EuiFlexGroup direction="column" gutterSize="m">
-        <EuiFlexItem>
-          <MultiSignalServicesTable
-            status={mainStatisticsStatus}
-            data={filteredData}
-            initialSortField={initialSortField}
-            initialPageSize={INITIAL_PAGE_SIZE}
-            initialSortDirection={INITIAL_SORT_DIRECTION}
-            timeseriesData={timeseriesDataFetch?.data}
-            timeseriesDataLoading={timeseriesDataFetch.status === FETCH_STATUS.LOADING}
-            noItemsMessage={
-              <EmptyMessage
-                heading={i18n.translate('xpack.apm.servicesTable.notFoundLabel', {
-                  defaultMessage: 'No services found',
+      {!data?.hasData && status === FETCH_STATUS.SUCCESS ? (
```

Comment:
I actually like more the way it was before. But no hard feelings.

Reply:
Me too but I had to render the Welcome modal.